docs/cross_toml.md

```diff
@@ -34,10 +34,26 @@ The `target` key allows you to specify parameters for specific compilation targe
 xargo = false
 build-std = false
 image = "test-image"
-pre-build = ["apt-get update"]
+pre-build = ["apt-get update"] # can also be the path to a file to run
 runner = "custom-runner"
 ```
 
+# `target.TARGET.pre-build`
+
```

Comment:
Right now it's on the host_root, defined as cargo manifest path or cwd if outside workspace. This is rather strange imo, but not sure how to make it as usable as possible.

Reply:
Yeah, I think it's important to document because this is the first situation where people without experience with Dockerfiles might need to know what a context is.